src/modules/ie-objects/controllers/ie-objects.controller.ts

```diff
@@ -170,12 +172,13 @@ export class IeObjectsController {
 		return convertObjectToCsv(objectMetadata);
 	}
 
-	@Get(':schemaIdentifier/related/:meemooIdentifier')
+	@Get(':schemaIdentifier/related/:meemooIdentifier/')
```

Comment:
REQ: don't add slash at the end of this endpoint. I presume you are having an issue getting to this endpoint when others take over. This is not the solution. 

Q: Can you explain why ending slash was added here?

Reply:
I first tried adding the new parameter trough the url but found it it didn't work. I thought it didn't matter whether the slash is there or not. I am gonna change it.

Reply:
Adding it to url would mean it is a **Param** but we need a **Query** here. So yes please remove.